lib-content-scrapers/src/jvmTest/kotlin/com/ustadmobile/lib/contentscrapers/folder/TestFolderIndexerAndScraper.kt

```diff
@@ -0,0 +1,119 @@
+package com.ustadmobile.lib.contentscrapers.folder
```

Comment:
Let's stick to the convention: this should be in two separate files: TestFolderIndexer and TestFolderScraper.